pages/windows/assoc.md

```diff
@@ -1,16 +1,16 @@
 # assoc
 
-> Display or modify file extension associations.
+> Display or change associations of file names.
```

Comment:
What is meant by an associations of file names?

Reply:
[How to associate a file with a program in Windows](https://www.computerhope.com/issues/ch000572.htm)

An association is smth that specifies which program should handle which file type.

Reply:
ah ok, so like the default applications each file type

Reply:
```suggestion
> Display each file type and the associated program which handles them:
```